### ISSUE TITLE
Unable to clip a screenshot but tray icon is there, no terminal output on why

### ISSUE THREAD

**Issue by 33Fraise33** [flameshot-org/flameshot#1395]
**Flameshot version**

flameshot-git-r1253.d9b9049-1
Flameshot v0.9.0
Compiled with Qt 5.15.2

**Describe the bug**
When opening trying to capture a screenshot either by using "flameshot gui" or by clicking or right clicking the tray icon and going to take screenshot doesn't show anything. 

**To Reproduce**

Running either: 
```
QT_QPA_PLATFORM=xcb flameshot
QSettings::value: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
```
or 
```
QT_QPA_PLATFORM=wayland-egl flameshot 
QSettings::value: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
QSettings::value: Empty key passed
QSettings::setValue: Empty key passed
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
```

**Expected behavior**

Shop the clip region allowing me to snap a screenshot

**System Information**

```
System:
  Host: T14-Fraise Kernel: 5.11.1-1-MANJARO x86_64 bits: 64 
  Desktop: KDE Plasma 5.21.1 Distro: Manjaro Linux 
Graphics:
  Device-1: AMD Renoir driver: amdgpu v: kernel 
  Device-2: IMC Networks Integrated Camera type: USB driver: uvcvideo 
  Display: wayland server: X.Org 21.0.99.1 driver: loaded: amdgpu,ati 
  unloaded: modesetting,vesa resolution: 1: 1920x1080~60Hz 2: 2560x1080~60Hz 
  OpenGL: renderer: AMD RENOIR (DRM 3.40.0 5.11.1-1-MANJARO LLVM 11.1.0) 
  v: 4.6 Mesa 20.3.4 
```

**Comment by 33Fraise33** [comment]
I just saw this in the wayland-session.log
.local/share/sddm/wayland-session.log

```
kwineffects: Process  9958  tried to take a screenshot without being granted to DBus interface "org.kde.kwin.Screenshot"
```

**Comment by borgmanJeremy** [comment]
This will be solved with #1380 .

If you want to test the fix listed there that would be helpful.

**Comment by 33Fraise33** [comment]
> This will be solved with #1380 .
> 
> If you want to test the fix listed there that would be helpful.

Thanks I will close this issue now, I commented over there
